app/views/books/show.html.erb

```diff
@@ -21,10 +21,15 @@
     <div class="row"><p><%= t('books.quantity', quantity: @book.quantity) %></p></div>
     <div class="row"><p><%= t('books.price', price: @book.price) %></p></div>
   </div>
-  <div class="col-sm-3 text-center text-big-size left-border">
-    <div class="row"><b><%= t('books.seller_name') %></b></div>
-    <div class="row"> <img src="https://images-na.ssl-images-amazon.com/images/I/71vntClRfjL._SX425_.jpg" class="user-avatar"> </div>
-    <div class="row"><p><%= "Thong tin cua nguoi ban neu user dang nhap" %></p></div>
+  <div class="col-sm-3 text-center left-border">
+    <div class="row text-big-size"><b><%= t('books.user.name', name: @book.user.name) %></b></div>
```

Comment:
Sử dụng `delegate`, ví dụ: https://coderwall.com/p/wamyow/using-delegate-in-rails